Source/Core/VideoBackends/Software/Rasterizer.cpp

```diff
@@ -71,6 +71,37 @@ void SetTevReg(int reg, int comp, s16 color)
   tev.SetRegColor(reg, comp, color);
 }
 
+// Does the depth test for all four pixels in the block
+// Returns true if at least one pixel passes
+static bool DepthTestBlock(s32 block_x, s32 block_y)
+{
+  int passCount = 0;
```

Comment:
I think this comment would benefit from something explicitly stating that this function also updates `pixel.mask` for all pixels in the block, as otherwise I wondered why it didn't return early as soon as one pixel passed.